.travis.yml

```diff
@@ -1,52 +1,32 @@
-language: cpp
-compiler:
-  - gcc
-sudo: false
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-      - boost-latest
-    packages:
-      - gcc-4.8
-      - g++-4.8
-      - libboost-filesystem1.55-dev
```

Comment:
I thought we needed boost 1.58 or greater?

Reply:
I don't remember where a requirement on 1.58 came from. That may have been a bug fix that wasn't necessary in CI, and getting newer versions in Travis was a pain.